backend/src/managers/TranslationManager.ts

```diff
@@ -163,7 +163,7 @@ export default class TranslationManager {
 
         // if parsingResult without html tags and whitespace is too short, don't do anything
         if (stripHtml(parsingResult).result.trim().length < 6) {
-            write(parsingResult);
```

Comment:
don't forget to remove debug

Reply:
?

Reply:
It just returns empty response.

Reply:
Ah, sorry - for some reason thought you've commited local debug code